docs/content/docs/start/tree-start.md

```diff
@@ -0,0 +1,239 @@
+---
+title: SharedTree Quick Start
+menuPosition: 2
+codeCopyButton: true
+aliases:
+  - "/docs/get-started/tree-quick-start/"
+  - "/start/tree-quick-start/"
+  - "/docs/tree-start/"
+
+---
+
+`SharedTree` is a [distributed data structure](../data-structures/overview.md) that looks and feels like simple JavaScript objects with a type safe wrapper.
+This guide will walk you through the basics of creating, configuring, and interacting with a `SharedTree` in your application. 
+
+## Creating a Tree
+
+A `SharedTree` can be created by defining a `ContainerSchema` with an initial object of type `SharedTree` and using this schema to create and load your container.
+This example creates a container using an Azure specific client.
+
+See more info on creating and loading containers [here](../build/containers.md#creating-a-container).
+
+```
+import { AzureClient } from "@fluidframework/azure-client";
+import { ContainerSchema, SharedTree } from "fluid-framework";
+
+// clientProps specifies connection details and is defined by the app author
+const client = new AzureClient(clientProps);
+
+const containerSchema: ContainerSchema = {
+	initialObjects: {
+		appData: SharedTree,
+	},
+};
+
+const { container } = await client.createContainer(containerSchema, "2");
+```
+
+After creating a `SharedTree`, you need to create a `TreeView`.
+A `TreeView` provides the interface for reading and editing data on the `SharedTree`.
+This is done by calling `viewWith` on the `SharedTree` with your tree configuration.
+
+```
+const treeConfiguration = new TreeViewConfiguration(
+	{ schema: RootSchema }
+)
+
+const appData = container.initialObjects.appData.viewWith(
+	treeConfiguration
+);
+```
+
+The tree configuration takes in a schema for the root of the tree which will need to be defined by your application.
+
+## Defining a Schema
+
+A schema outlines the structure and types of data that your tree will manage.
+Having a schema provides useful guarantees about the shape your data will take.
+It is also used to generate TypeScript types which means the structure of your data is analogous to JS data types for easy use in external libraries or systems.
```

Comment:
nit: the generation of types is not what ensures the structure/usefulness and external systems, so you could rework this sentence and the one before it for more clarity